### PR TITLE
Scenario tutorials

### PR DESCRIPTION
**Pull request by MObrien-IDM**
### Brief description
Creating new tutorials for the scenarios.py script. 
T3 - building and running multiple scenarios
T4 - plotting options for scenarios

Addresses issue #61 by removing legacy script, issue #40 by developing new tutorial, issue #49 which overlaps 40. 

Note: I find it easier to review the jupyter notebooks by navigating to the branch and taking a look at the files in the github preview panes, rather than staring at the raw code in the PR itself. YMMV!

### Type(s) of change
- [ ] Bugfix
- [ ] Refactor
- [x] New feature
- [ ] Other

### Checklist
- My code follows the [style guide](https://github.com/amath-idm/styleguide)
  - [x] Yes
  - [ ] N/A
- I've commented my code
  - [x] Yes
  - [ ] N/A
- I've incremented the version number
  - [ ] Yes
  - [x] N/A
- I've updated the changelog
  - [ ] Yes
  - [x] N/A
- I've added tests and checked code coverage
  - [ ] Yes
  - [x] N/A

### CODE REVIEW

File: docs/tutorials/T1_intro.ipynb [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "- Running a simulation\n",
     "- Plotting results\n",
     "\n",
-    "This tutorial walks you through how to define paramaters and run the simulation. The next tutorial will show you how to plot the results of a simulation."
+    "This tutorial walks you through how to define parameters and run the simulation. The next tutorial will show you how to plot the results of a simulation."
```
Comment (mzimmermann-IDM) [inline docs/tutorials/T1_intro.ipynb:20]: Line 28: heading should be changed to something like "import FPsim"
Line 57-62: needs some punctuation or formatting
Line 64 typo 'these'
Line 148: can you add a description of what this line does?

File: docs/tutorials/T3_scenarios.ipynb [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+    "                )\n",
+    "    f_switch = fp.make_scen(\n",
+    "            label = '20 percent switching',\n",
+    "            year = year,\n",
+    "            probs = [\n",
+    "                dict(source = 'Injectables', value = 0.20, **d_kw) #note the use of **d_kw here\n",
+    "                    ]\n",
+    "               )\n",
+    "\n",
+    "    f4 = f2 + f_switch # We wil come back to this later.\n",
```
Comment (mzimmermann-IDM) [inline docs/tutorials/T3_scenarios.ipynb:189]: Typo will

**Review by mzimmermann-IDM — COMMENTED** [review]
FYI in T2 there's a CK comment in the as_cpr plot.

This looks great! Thanks for writing it up.

**Review by cliffckerr — COMMENTED** [review]
Looks good! Minor comments:
- For function arguments, there shouldn't be spaces (e.g. `scens.add_scen(f4, label='5x uptake plus switching')`, not `scens.add_scen(f4, label = '5x uptake plus switching')`)
- Probably better to link to the [API docs](https://docs.idmod.org/projects/fpsim/en/latest/fpsim.scenarios.html#fpsim.scenarios.Scenario) rather than copy-paste. No way we're going to remember to update them!
- T3 feels pretty complicated to me. It's complete, but I feel it might be a bit heavy for users who just want to get a taste.
- I think T4 can be merged with T3, especially if T3 can be shortened a little.

File: docs/tutorials/T1_intro.ipynb (hunk 1 above)
Comment (avictorious) [reply docs/tutorials/T1_intro.ipynb:20]: I'm having a hard time finding the line numbers in the notebook, so these are likely duplicate comments.
This line needs some clarity or formatting.  Not sure I understand the parameter categories as you lay them out
- Parameters are defined as a dictionary. In FPsim, we categorize our parameters as:
Basic parameters Age limits Durations Pregnancy outcomes Fecundity and exposure MCPR

**Review by avictorious — COMMENTED** [review]
Also, in T1, the last line of the notebook isn't clear what it's being used for: fp.__file__

**Pull request merged**